### ISSUE TITLE
Show item counts behind filters

### ISSUE THREAD

**Issue by d4rken** [d4rken-org/mystic-permissions#11]
3. Can you include count of each category in brackets (behind the filter names)?

_Originally posted by @TheMysticS in https://github.com/d4rken/mystic-permissions/issues/1#issuecomment-1186378156_

**Comment by d4rken** [comment]
Out of scope for now as we know the item code only after filtering, so to show the item codes in the dialog we would have to apply each filter. This would cause a significant delay before we could show the dialog, not worth it imho.

**Comment by TheMysticS** [comment]
Every time the app is launched or when pressing the refresh/ rescan button, does the app not perform a count of these categories in the background?

**Comment by d4rken** [comment]
No, the app generates a list of all apps, then when a filter is select the list displayed in the UI will be adjusted.

**Comment by TheMysticS** [comment]
> No, the app generates a list of all apps, then when a filter is select the list displayed in the UI will be adjusted.

Can the app not be made to do this computation in the background even without the user selecting any filter?

Just like how SD Maid shows the count of number of apps that are Stopped, Frozen, Boot, etc.?

**Comment by d4rken** [comment]
I'll take another look

**Comment by d4rken** [comment]
Cost/benefit is too low.
